modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -1227,6 +1228,43 @@ Suggestions for improvement::
 Use `DATABASE *` without the parameter to revoke the privilege on all databases.
 ====
 
+[#_topology_notifications]
+== `TOPOLOGY` category
+
+Topology notifications provide additional information related to managing databases and servers.
+databases and/or servers has some additional information to tell the user.
```

Comment:
Now this line start mid+sentence, should it be removed as well?

Reply:
Already done! :smile: